### ISSUE TITLE
After update FreshRSS update to 1.23.1, FeedPriorityShortcut throws PHP Fatal error

### ISSUE THREAD

**Issue by p37307** [aidistan/freshrss-extensions#5]
FreshRSS updated to 1.23.1.

Went to Subscriptions page ( _/p/i/?c=subscription_ ). It was blank.

Looked at server logs, it showed errors in this extension. Disabled extension and everything works correctly.

I checked the extension edge version and it appears to be the same as I am running.

**Extensions metadata.json**
```
{
    "name": "Feed Priority Shortcut",
    "author": "Aidi Stan",
    "description": "Set up visibilities/priorities of your feeds easily",
    "version": "1.0.3",
    "entrypoint": "FeedPriorityShortcut",
    "type": "user"
}
```

**Web Server Error Log:**

```
[20-Jan-2024 09:22:50 America/New_York] PHP Fatal error:  Uncaught Error: Undefined constant FreshRSS_Feed::PRIORITY_NORMAL in /home/REDACTED/FreshRss/extensions/xExtension-FeedPriorityShortcut/extension.php:34
Stack trace:
#0 [internal function]: FeedPriorityShortcutExtension->{closure}()
#1 /home/REDACTED/FreshRss/extensions/xExtension-FeedPriorityShortcut/extension.php(37): array_map()
#2 /home/REDACTED/FreshRss/lib/Minz/ExtensionManager.php(374): FeedPriorityShortcutExtension->provideFeedPrioritiesInJS()
#3 /home/REDACTED/FreshRss/lib/Minz/ExtensionManager.php(344): Minz_ExtensionManager::callOneToOne()
#4 /home/REDACTED/FreshRss/app/views/helpers/javascript_vars.phtml(6): Minz_ExtensionManager::callHook()
#5 /home/REDACTED/FreshRss/lib/Minz/View.php(93): include('/home/REDACTED/...')
#6 /home/REDACTED/FreshRss/lib/Minz/View.php(143): Minz_View->includeFile()
#7 /home/REDACTED/FreshRss/app/layout/layout.phtml(20): Minz_View->renderHelper()
#8 /home/REDACTED/FreshRss/lib/Minz/View.php(93): include('/home/REDACTED/...')
#9 /home/REDACTED/FreshRss/lib/Minz/View.php(106): Minz_View->includeFile()
#10 /home/REDACTED/FreshRss/lib/Minz/View.php(73): Minz_View->buildLayout()
#11 /home/REDACTED/FreshRss/lib/Minz/Dispatcher.php(59): Minz_View->build()
#12 /home/REDACTED/FreshRss/lib/Minz/FrontController.php(61): Minz_Dispatcher->run()
#13 /home/REDACTED/FreshRss/p/i/index.php(59): Minz_FrontController->run()
#14 {main}
  thrown in /home/REDACTED/FreshRss/extensions/xExtension-FeedPriorityShortcut/extension.php on line 34

[20-Jan-2024 09:22:51 America/New_York] PHP Fatal error:  Uncaught Error: Undefined constant FreshRSS_Feed::PRIORITY_NORMAL in /home/REDACTED/FreshRss/extensions/xExtension-FeedPriorityShortcut/extension.php:34
Stack trace:
#0 [internal function]: FeedPriorityShortcutExtension->{closure}()
#1 /home/REDACTED/FreshRss/extensions/xExtension-FeedPriorityShortcut/extension.php(37): array_map()
#2 /home/REDACTED/FreshRss/lib/Minz/ExtensionManager.php(374): FeedPriorityShortcutExtension->provideFeedPrioritiesInJS()
#3 /home/REDACTED/FreshRss/lib/Minz/ExtensionManager.php(344): Minz_ExtensionManager::callOneToOne()
#4 /home/REDACTED/FreshRss/app/views/helpers/javascript_vars.phtml(6): Minz_ExtensionManager::callHook()
#5 /home/REDACTED/FreshRss/lib/Minz/View.php(93): include('/home/REDACTED/...')
#6 /home/REDACTED/FreshRss/lib/Minz/View.php(143): Minz_View->includeFile()
#7 /home/REDACTED/FreshRss/app/layout/layout.phtml(20): Minz_View->renderHelper()
#8 /home/REDACTED/FreshRss/lib/Minz/View.php(93): include('/home/REDACTED/...')
#9 /home/REDACTED/FreshRss/lib/Minz/View.php(106): Minz_View->includeFile()
#10 /home/REDACTED/FreshRss/lib/Minz/View.php(73): Minz_View->buildLayout()
#11 /home/REDACTED/FreshRss/lib/Minz/Dispatcher.php(59): Minz_View->build()
#12 /home/REDACTED/FreshRss/lib/Minz/FrontController.php(61): Minz_Dispatcher->run()
#13 /home/REDACTED/FreshRss/p/i/index.php(59): Minz_FrontController->run()
#14 {main}
  thrown in /home/REDACTED/FreshRss/extensions/xExtension-FeedPriorityShortcut/extension.php on line 34
```

**Comment by aidistan** [comment]
@p37307 Thanks for reporting the issue. It should be fixed in FeedPriorityShortcut v1.1.0

**Comment by p37307** [comment]
> @p37307 Thanks for reporting the issue. It should be fixed in FeedPriorityShortcut v1.1.0

@aidistan  I can confirm that v1.1.0 fixed this issue I reported. Thanks for the quick repair.
